### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by pasha1932**
- [DEMO LINK](https://pasha1932.github.io/layout_catalog/)
- [TEST REPORT LINK](https://pasha1932.github.io/layout_catalog/report/html_report/)

❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.

- [x] All component follow BEM and use SCSS
- [x] repaeted sizes and special colors are put to variables
- [x] Grid is used for the columns
- [x] cards are shown in 1, 2, 3 or 4 columns based on screen resolution
- [x] All changes on `:hover` are smooth
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
+.catalog {
```
Comment (maxim2310) [inline src/styles/blocks/card.scss:1]: need to move it to separate file as BEM block

File: src/styles/blocks/star.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
```
Comment (maxim2310) [inline src/styles/blocks/star.scss:32]: combine it like 
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
....

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
Good job, but now, when I hover over the "Buy" button, elements in the below row gets distracted

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
```
Comment (sTorba24) [inline src/styles/blocks/header.scss:11]: move this color to variable

**Review by sTorba24 — APPROVED** [review]

**Pull request closed without merging**